app/services/induction/review_cohort_after_eligibility_checks.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Change the cohort of an ECF participant whose eligibility has been recalculated.
+# If the participant has become eligible and is currently in a payments-frozen cohort,
+# transfer them to the currently active registration cohort.
+module Induction
+  class ReviewCohortAfterEligibilityChecks
```

Comment:
This service name doesn't really suggest to me that anything will be updated here. I think we should be more explicit about the fact records are likely to change in this service.
Something like `UpdateCohortAfterEligibilityChecks` or `AmendCohortAfterEligibilityChecks` ?

Reply:
Yes, the thing is that the cohort might be or might not be changed. So that's why I didn't use Amend or Update.

Reply:
Anyway. Renamed